### PR TITLE
Update readme and use correct url schema in build

### PR DESCRIPTION
**Pull request by brunovesar**
After the meeting yesterday, I reviewed the readme and found some things to fix the issues I was having before and I mentioned yesterday.
I updated the readme to make it easier to follow the setup steps, and added a troubleshooting section.
Also fixed the url schema in the build.sh and added the chmod to the data_loader.

### CODE REVIEW

**Review by sofia-doofinder — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,9 +12,17 @@
 dos2unix build.sh
 ```
 
-Rename the `.env.example` to `.env` and set your tokens provided by magento at the [admin panel](https://commercemarketplace.adobe.com/customer/accessKeys) in the `COMPOSER_AUTH` environment variable. You can also set the version of Magento you wish to install.
+**Configure NGROK**
+In order to be able to create an account or login to an existing Doofinder account during the module initial setup, you will have to expose your local webserver to internet (to receive a callback).
+
+To do so, you can use, for example; the utility Ngrok: https://dashboard.ngrok.com/get-started/setup
```
Comment (davidmolinacano) [inline README.md:18]: FYI, we have also a custom ngrok that uses a *.doofinder.com subdomain: https://github.com/doofinder/dfcommon/wiki/Doongrok

**Review by davidmolinacano — APPROVED** [review]
Excellent work!! :clap: :medal_military: :100:

File: README.md (hunk 1 above)
Comment (brunovesar) [reply README.md:18]: Sure!
But as this is a public repo I didn't want to mention private tools

Comment (davidmolinacano) [reply README.md:18]: Good point! Then forget about what I said before :face_in_clouds:

**Pull request merged**